spring/src/main/java/org/axonframework/spring/config/SpringAggregateConfigurer.java

```diff
@@ -160,7 +165,19 @@ public void configureModule(@Nonnull Configurer configurer) {
             aggregateConfigurer.configureRepository(
                     c -> applicationContext.getBean(aggregateRepository, Repository.class)
             );
+        } else if (isEntityManagerAnnotationPresent(aggregateType)) {
```

Comment:
This was copied from the 4.5.x I believe, and is missing the `.spanFactory(SpanFactory)` configuration. Are we sure we didn't miss anything else?

Reply:
Just validated the builder, but this is all we ever auto-configured for the `GenericJpaRepository`.
You're right on the `SpanFactory` though, we should include that.